packages/cli/src/databases/entities/User.ts

```diff
@@ -113,4 +113,16 @@ export class User extends AbstractEntity implements IUser {
 	computeIsPending(): void {
 		this.isPending = this.password === null;
 	}
+
+	/**
+	 * Whether the user is instance owner
+	 */
+	isOwner: boolean;
+
+	@AfterLoad()
+	computeIsOwner(): void {
+		// If globalRole is not defined, it means that the owner is not set-up yet
+		this.isOwner =
+			this.globalRole === undefined ? this.globalRoleId === '1' : this.globalRole.name === 'owner';
```

Comment:
Is it safe to consider a shell owner and a fully set-up owner equal under `user.isOwner`? That is, if the instance owner has not been set up yet, is it correct to treat them as the instance owner already? None of the checks care about this? Wondering if this could have unintended side effects or turn out to be misleading for others relying on this in future.

Side note: I'd love if we could find a way to avoid relying on a specific ID, as this can be a bit brittle, unless too time-consuming.

Reply:
I agree that it's not ideal to be checking for a specific role ID but I couldn't find a better way. So I reverted back to the original implementation `this.isOwner = this.globalRole?.name === 'owner';`.